src/components/Lend/LoanSearch/ChallengeHeader.vue

```diff
@@ -153,6 +153,11 @@ export default {
 			return this.challengeData?.participation ?? {};
 		}
 	},
+	mounted() {
+		this.$nextTick(() => {
```

Comment:
Wondering why nextTick?

Reply:
Right, but why is it needed instead of just called in `mounted` directly?